### DISCUSSION TITLE
TUDaThesis - custommargins

### DISCUSSION THREAD

**Discussion by al-bonin** [tudace/tuda_latex_templates#371] [Q&A] [answered]
Hallo, ich habe die Anleitungen und geschloßene Tickets über custommargins in TUDaThesis gelesen. Ich bin jetzt total verwirrt, vielleicht liegt es an meiner Sprache. 

Ich habe das so verstanden, dass "custommargins=False" die richtige Corporate Design Einstellung für die Thesis ist, ist das richtig? Obwohl es standardmäßig aktiviert(True) ist, darf man sie deaktivieren? 

Vielen Dank im Voraus und Schönes Wochenende
Ercan

**Comment by TeXhackse** [comment, marked as answer]
Hallo,
ich veruche mal zu erklären:

`custommargins=false` ist der default, den tudapub liefert. Die Randeinstellungen entsprechen dann den offiziellen Vorgaben. Diese Ränder sind aber auch für ein paar anderen Anpassungen gedacht, z.B. die Randnotizspalte und eine andere Schriftgröße.

Bei der Erstellung der Templates haben wir beschlossen, dass es sinnvoller ist eine Basiseinstellung für die Thesis-Templates zu setzen die – im Sinne der besseren Lesbarkeit – eine kürzere Zeilenlänge und eine größere Schriftgröße nutzt. Wir nutzen hierfür die Satzspiegelberechnung mit typearea vgl. KOMA-Script Doku (https://texdoc.org/serve/scrguide/0#page=27)
 
Es gibt (leider) keine Universitätsweiten Vorgaben für die Ränder. Demnach können wir ohnehin keine allgemeingültige Lösung liefern sondern nur dabei unterstützen mögliche Vorgaben umzusetzen. 

Daher sind auch alle Möglichkeiten zu den Anpassungen der Ränder dokumentiert.

Es ist also aus unserer Perspektive alles erlaubt. Es kann allerdings sein, dass es Fachbereichsspezifische Vorgaben gibt, von denen wir nichts wissen. 

`custommargins=false` sind einfach aus typografischer Sicht für einspaltife Texte zu lage Zeilenlängen. Deswegen weicht hier der Default von den Vorgaben ab.

Ich hoffe das ist so etwas besser verständlich, ich werde die Doku diesbezüglich nochmal durchgehen, ob ich es dort vielleicht klarer Formulieren kann und einen Wiki Artikel ergänzen.
